pennylane/math/__init__.py

```diff
@@ -70,6 +70,7 @@
 from .is_independent import is_independent
 
 sum = ar.numpy.sum
+mult = ar.numpy.multiply
```

Comment:
Is there a reason to not just use `qml.math.multiply`? 

If at all possible, it's best to keep the name of the function from the other packages as that is what users and developers will be familiar with and expect.

Reply:
I think since we needed an explicit add method I also added the multiply method. I will swap over to `qml.math.multiply` but if the function needs additional logic (like in the case of add) we can update this 👍🏼